### ISSUE TITLE
June 14 2023 Update

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2273]
**Broken Link Checker**
- waiting on update see #2216 

**Events Manager 6.3 -> 6.4.1**
- [Release Notes](https://wordpress.org/plugins/events-manager/#developers)
- looks like they made a bunch of updates to the booking forms so if we have a customer that uses Events Manager to book events we should check it out. If all event managment is done in the WP admin then I don't see anything to check.

**Gravity Forms 2.7.6 -> 2.7.8**
- [Release Notes](https://docs.gravityforms.com/gravityforms-change-log/)
- they did something to change how forms render that deals with the shortcode. We use the shortcode all the time because we're using the old WP editor. We need to check that forms are rendering as expected

**mailoptin 2.2.58.2 -> 2.2.59.0**
- [Release Notes](https://mailoptin.io/changelog/)
- I don't see anyhting that should affect us

**SiteOrigin Panels 2.22.1 -> 2.23.0**
- [Release Notes](https://wordpress.org/plugins/siteorigin-panels/#developers)
- says they make a minor style update to the page builder settings pages. Need to check if it matches still with our theme

**Post Expirator 2.9.2 3.0.0**
- [Release Notes](https://wordpress.org/plugins/post-expirator/#developers)
- they changed the way the schedule the post expiration actions. They're using code from WooCommerce so it's good code, but we should still double check that items are expiring as expected
- they made some minor text changes to labels. We should evaluate any communication that may need to be made with customers

**Safe Redirect Manager 1.11.1 -> 2.0.1**
- [Release Notes](https://wordpress.org/plugins/safe-redirect-manager/#developers)
- nothing should impact us it was bug fixes and developer quality of life updates

**Google Site Kit 1.99.0 -> 1.102.0**
- [Release Notes](https://wordpress.org/plugins/google-site-kit/#developers)
- looks like bug fixes so nothing should impact us. We likely need to deploy to ProudCity first and test there since it has active analytics

**SO Widgets Bundle 1.50.0 -> 1.50.1**
- [Release Notes](https://wordpress.org/plugins/so-widgets-bundle/#developers)
- They update Font Awesome from 6.2.0 -> 6.4.0 we should see if that impact us and likely add an issue to upgrade ourselves

**WP Job Manager 1.40.1->1.40.2**
- [Release Notes](https://wordpress.org/plugins/wp-job-manager/#developers)
- bug fix shouldn't impact us

**WP Media Folder 5.5.10 -> 5.5.11**
- [Release Notes](https://www.joomunited.com/changelog/wp-media-folder-changelog)
- I don't see anything, but their release notes are fairly vague

**WordPress SEO 20.7 -> 20.9**
- [Release Notes](https://wordpress.org/plugins/wordpress-seo/#developers)
- nothing should affect us though if we're using the standard WordPress search version 20.9 [adds work](https://yoast.com/yoast-seo-june-13-2023/#shortlink=release-13-6-23) to remove the default WP search pages from any SEO indexing which can stop page clutter in our Google Search results and indexing

## Curtis Checklist

- [x] update and deploy to beta
- [x] check events manager forms (if needed)
- [x] check that GF forms render as expected
- [x] check Site Origin Panels settings page and make sure the styles are still appropriate
- [x] verify Post Expirator expires posts as expected
- [x] deploy to ProudCity and verify that Site Kit works as expected
- [x] check SO Widgets Bundle Font Awesome
- [x] create issue to upgrade our Font Awesome library from 6.2.0 -> 6.4.0
  - see: https://github.com/proudcity/wp-proudcity/issues/2278

**Comment by curtismchale** [comment]
@kevindherman given the above updates:

- do we have anyone that lets users register events with the build in Events Manager forms? If they use Gravity Forms we shouldn't need to worry about it.

**Comment by kevindherman** [comment]
@curtismchale the feature is a bit buried with older sites and something we've moved away from. Having said that, it's hard to know who is actually using it or not.

My hunch (and hope) is barely anyone. There's an "add event" icon on old sites in the event sidebar that's the only way to get to the feature. If that's removed and the add event page is removed then we can fully move off of the legacy option.

**Comment by curtismchale** [comment]
@kevindherman @lukefretwell so the only "issue" I see is that Font Awesome has introduced new icons that we don't support currently. We'd need to run an upgrade cycle to get them in as we build the list of available icons and then push it out to customers once we've upgraded. I'll create an issue specifically for this.

**Comment by kevindherman** [comment]
@curtismchale I did notice that there may be more icon options. I think it's smart to put into a separate issue for the next release since that can take more testing. cc @lukefretwell

**Comment by lukefretwell** [comment]
Looks good to me @curtismchale @kevindherman. I mostly tested the broken link checker. Love the format of this issue @curtismchale .
